modules/ROOT/pages/browser-settings.adoc

```diff
@@ -5,16 +5,31 @@ endif::[]
 
 Anypoint Studio comes bundled with a default browser to display elements of the Anypoint Platform UI. For example, when you deploy to CloudHub from Studio, the Runtime Manager UI displayed in Studio is rendered using a built-in browser.
 
-This built-in browser is the default. However, if it fails to display, you can configure Studio to use your system's native browser to render the Anypoint Platform UI:
+This built-in browser is the default. However, if you log in to Anypoint Platform using multi-factor authentication (MFA), you must configure Studio to use your system's native browser to render the Anypoint Platform UI:
 
 . Navigate to *Preferences* > *Anypoint Studio* > *Browser*.
 . Select the type of default browser for Studio:
 +
 image::browser-configuration.png[]
-<1> *Built-in*: Uses the built-in browser. This is the recommended setting.
+<1> *Built-in*: Uses the built-in browser.
 <2> *Native*: Uses the native browser of your operative system.
 . Click *Preview* to verify the changed configuration. +
 
+Additionally, if you use Windows and your default browser is Internet Explorer 11, you must perform an additional configuration:
```

Comment:
I suggest to remove "and your default browser is Internet Explorer 11" because it really don't matter what's the default browser configured by the user. Studio's native browser will always use IE (9 or 11) and both fail.